src/pages/workspaces/List.js

```diff
@@ -167,7 +167,7 @@ export const WorkspaceList = () => {
   const [cloningWorkspaceId, setCloningWorkspaceId] = useState();
   const [deletingWorkspaceId, setDeletingWorkspaceId] = useState();
   const [lockingWorkspaceId, setLockingWorkspaceId] = useState();
-  const [sharingWorkspaceId, setSharingWorkspaceId] = useState();
```

Comment:
I switched to using the entire workspace in the state so the component could pass the policies to the modal, without dealing with modifying the loaded workspaces directly.